training/cloud/README.md

```diff
@@ -16,7 +16,7 @@ Please refer to the official RHEL AI documentation on how to create machine imag
 
 | Variable                  | Description                                     | Default                                                      |
 |---------------------------|-------------------------------------------------|--------------------------------------------------------------|
-| CLOUD                     | Sets the name of the cloud: aws, gcp, azure, ...| ` `                                                          |
+| CLOUD                     | Sets the name of the cloud: aws, gcp, ibm, ...  |  ` `                                                          |
```

Comment:
Why are we removing the `azure` entry, rather than just adding a new option for IBM? I still see azure listed [in source](https://github.com/containers/ai-lab-recipes/tree/main/training/cloud/azure) which is confusing me, please either do not remove it from these docs, or remove it from source.